common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -231,6 +231,91 @@ describe 'Problem', ->
       runs ->
         expect(@problem.checkButtonLabel.text).toHaveBeenCalledWith 'Check'
 
+  describe 'impossible to click "check" without selecting a choice"', ->
+    beforeEach ->
+      @problem = new Problem($('.xblock-student_view'))
+      @checkDisabled = (v) -> expect(@problem.checkButton.hasClass('is-disabled')).toBe(v)
+
+    describe 'some basic tests', ->
+      it 'test that check button in disabled by default', ->
```

Comment:
In the Jasmine test framework, the names `describe` and `it` aren't accidental. Ideally, you should name your tests based on what it is you're testing. In this case, it sounds like you want something like:
```coffee
describe 'check button on problems', ->
  it 'should be disabled by default', ->
    // ...
  it 'should become enabled after a value is entered into the text box', ->
    // ...
  it 'should become enabled after a checkbox is checked', ->
    // ...
  it 'should become enabled after a value is selected in a selector', ->
    // ...
```
and so on.